contrib/k8s/examples/mysql/mysqldb-all-in-one-binding.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: servicecatalog.k8s.io/v1beta1
 kind: ServiceBinding
 metadata:
-  name: my-mysqldb-binding
+  name: my-mysql-binding
```

Comment:
I want to suggest that across the board, we make the names of the manifest files match the names of the resources contained therein.

Reply:
Argh yeah, forgot to rename the files when i moved them and dropped the db to match. Fixed that.